README.md

```diff
@@ -1,84 +1,273 @@
-# Node.js Toolkit
-
-#### The toolkit is a Node.js wrapper over the XMLSERVICE open source project from IBM. 
+# Node.js iToolkit <!-- omit in toc -->
 
 [![NPM](https://nodei.co/npm/itoolkit.png?downloads=true&downloadRank=true)](https://nodei.co/npm/itoolkit/)
 
-# Installation  
 
-Installation is done from a PASE shell.
+`itoolkit` is a Node.js interface to [XMLSERVICE](https://github.com/IBM/xmlservice) to access all things IBM i.
+
+# Table of Contents <!-- omit in toc -->
+- [Installation](#installation)
+  - [Main Classes](#main-classes)
+    - [Connection](#connection)
+      - [Transports](#transports)
+        - [idb-connector](#idb-connector)
+        - [REST](#rest)
+        - [SSH](#ssh)
+    - [ProgramCall](#programcall)
+      - [Example](#example)
```

Comment:
@ThePrez Each of the basic api examples were moved under the appropriate type.

For example the ProgramCall example was moved here